CONTRIBUTING.rst

```diff
@@ -1,13 +1,12 @@
 .. SPDX-License-Identifier: MIT OR Apache-2.0
    SPDX-FileCopyrightText: The Ferrocene Developers
+   SPDX-FileCopyrightText: The Rust Project Contributors
 
 ====================================================
-Contributing to the Ferrocene Language Specification
+Contributing to the FLS
 ====================================================
 
-We released the Ferrocene Language Specification (FLS) publicly under an open
-source license. That's always been our goal, and since we announced our effort,
-a lot of people and organizations expressed their interest in it!
+The FLS is released publicly under an open source license, specifically MIT and Apache 2.0. 
```

Comment:
There is still trailing whitespace (i.e. spaces that follow the last visible character) on this line and elsewhere in this diff.  Also, this line needs to be wrapped.